src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -997,7 +997,12 @@ REDHAWK_PALEXPORT UInt32_BOOL REDHAWK_PALAPI PalRegisterHijackCallback(_In_ PalH
 
 REDHAWK_PALEXPORT void REDHAWK_PALAPI PalHijack(HANDLE hThread, _In_opt_ void* pThreadToHijack)
 {
-    ThreadUnixHandle* threadHandle = (ThreadUnixHandle*)hThread;
+// TODO-LLVM: enable if we implement wasm threads
+#if HOST_WASM
+    abort();
+#endif
+
```

Comment:
Do we need this abort or can we leave this code alone to be deleted by the compiler?
```suggestion
```

Reply:
We can remove it, thanks